build-system/pr-check/visual-diff-tests.js

```diff
@@ -42,7 +42,10 @@ function prBuildWorkflow() {
   if (buildTargetsInclude(Targets.RUNTIME, Targets.VISUAL_DIFF)) {
     downloadNomoduleOutput();
     timedExecOrDie('gulp update-packages');
-    timedExecOrDie('gulp visual-diff --nobuild');
+    // TODO(#30291): switch to prod config once FF launches aspect-ratio.
+    // See #32536 for more info.
+    // See https://bugzilla.mozilla.org/show_bug.cgi?id=1528375.
+    timedExecOrDie('gulp visual-diff --nobuild --config=no');
```

Comment:
If this works, we'll need to do the same thing for push builds too (line 37).

Reply:
Oh wait, I just realized that since we're passing in `--nobuild` to `gulp visual-diff`, the value of `--config` is a no-op. This is because we're downloading binaries that were built during a previous job, and that binary will have the prod config.

Hopefully the approach in #32537 works, in which case that is a preferable approach to the one in this PR.